src/package/packageVersion.ts

```diff
@@ -336,6 +343,43 @@ export class PackageVersion {
     }
   }
 
+  /**
+   * Query the Package2Version SObject and return data with the provided type.
+   *
+   * NOTE: There is a limit of 2000 records that can be returned, otherwise
+   * a GACK might be thrown.
+   *
+   * @param connection jsForce Connection to the org.
+   * @param fields The fields to include in the returned data. Defaults to all fields.
+   * @param whereClause Optional where clause to filter the query.
+   * @param whereClauseItems Optional array of where clause items to match. The query is chunked,
+   * meaning broken into multiple queries when the query length would exceed the maximum char limit.
+   * When defining items here, the `whereClause` argument must use this token for the item
+   * replacement: `'%IDS%'`.
+   * @returns Results from querying the Package2Version SObject.
+   */
+  public static async queryPackage2Version<T extends Record<string, unknown> = Record<string, unknown>>(
+    connection: Connection,
+    fields = Package2VersionFields,
+    whereClause?: string,
+    whereClauseItems?: string[]
+  ): Promise<T[]> {
+    let query = `SELECT ${fields.toString()} FROM Package2Version`;
+    if (whereClause) {
+      query += ` ${whereClause}`;
+      if (whereClauseItems) {
+        query += ' LIMIT 2000';
+        return queryWithInConditionChunking<T>(query, whereClauseItems, '%IDS%', connection);
+      }
+    }
+    query += ' LIMIT 2000';
```

Comment:
when using SOQL `limit`: is there a `order by` that makes it more likely the user gets relevant results?
ex: `order by lastModifiedDate DESC`?